tools/oeutil/host/generate_evidence.cpp

```diff
@@ -90,6 +322,16 @@ void decode_certificate_pem(FILE* file, const uint8_t* data, size_t data_len)
     BIO_free_all(input);
 }
 
+void decode_crl_der(const uint8_t* data, size_t data_len)
+{
+    X509_CRL* x509;
+    BIO* input = BIO_new_mem_buf(data, (int)data_len);
```

Comment:
```suggestion
    BIO* input = BIO_new_mem_buf(data, (int)data_size);
```

Avoid abbreviations, per OE style guidance